lib/eex/lib/eex/tokenizer.ex

```diff
@@ -1,6 +1,13 @@
 defmodule EEx.Tokenizer do
   @moduledoc false
 
+  @type marker :: '=' | ''
+  @type contents :: IO.chardata
+  @type line :: non_neg_integer
+  @type token :: {:text, contents} |
+                 {:expr | :start_expr | :middle_expr | :end_expr, line, marker, contents}
+  @type error :: String.t
```

Comment:
Since those types are used in the documentation for `tokenize/3`, I think it's easier to leave the types as they are. We could get rid of `error` and `line`, but it would leave `marker`.